### PR TITLE
UIDATIMP-514: SyntaxError: invalid regexp group in FF :: Fixed

### PR DESCRIPTION
**Pull request by IhorKoba**
## Approach
* Get rid of lookbehind usage in regular expression

## Ticket
[UIDATIMP-514](https://issues.folio.org/browse/UIDATIMP-514)

### CODE REVIEW

File: src/utils/updateValueWithTemplate.js [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 
   let updatedString = template;
   const match = template.match(findPattern);
 
   if (!isEmpty(match)) {
     match.forEach(key => {
-      updatedString = updatedString.replace(replacePattern, option[key]);
+      const currentKey = key.replace(/\*\*/g, '');
+      const replacePattern = new RegExp(`\\**${currentKey}\\**`, 'g');
+      const templateHasOption = !!template.match(currentKey);
```
Comment (mariia-aloshyna) [inline src/utils/updateValueWithTemplate.js:13]:
```suggestion
      const templateHasOptions = !isEmpty(template.match(currentKey));
```

File: src/utils/updateValueWithTemplate.js [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
+  const findPattern = /\*\*(\w+)\*\*/g;
 
   let updatedString = template;
   const match = template.match(findPattern);
 
   if (!isEmpty(match)) {
     match.forEach(key => {
-      updatedString = updatedString.replace(replacePattern, option[key]);
+      const currentKey = key.replace(/\*\*/g, '');
+      const replacePattern = new RegExp(`\\**${currentKey}\\**`, 'g');
```
Comment (mariia-aloshyna) [inline src/utils/updateValueWithTemplate.js:12]: Are you sure you need escape characters here?

Comment (IhorKoba) [reply src/utils/updateValueWithTemplate.js:12]: according to developer.mozilla.org, yes - 
image

Comment (mariia-aloshyna) [reply src/utils/updateValueWithTemplate.js:12]: In this case, maybe it's needed to escape second asterisk as well?
```suggestion
      const replacePattern = new RegExp(`\\*\\*${currentKey}\\*\\*`, 'g');
```

Comment (IhorKoba) [reply src/utils/updateValueWithTemplate.js:12]: I've tested and it works the same in both cases..

Comment (mariia-aloshyna) [reply src/utils/updateValueWithTemplate.js:12]: Did you try to match smth like this - `****code*****`?

Comment (IvanKryzhanovskyi) [reply src/utils/updateValueWithTemplate.js:12]: should we really cover such case?

Comment (IhorKoba) [reply src/utils/updateValueWithTemplate.js:12]: you're right, not the same. escaped second asterisk as well.

File: src/utils/updateValueWithTemplate.js (hunk 1 above)
Comment (IhorKoba) [reply src/utils/updateValueWithTemplate.js:13]: What is the difference, except one more method is used ?

Comment (mariia-aloshyna) [reply src/utils/updateValueWithTemplate.js:13]: As for me, it looks a little bit more readable, but it's up to you actually

Comment (IhorKoba) [reply src/utils/updateValueWithTemplate.js]: ok

**Pull request merged**